.github/workflows/ci-pr.yml

```diff
@@ -26,18 +26,10 @@ jobs:
         shell: bash
         run: find . -type f -name "*.sh" -exec chmod a+x '{}' \;
       - name: 'Configure: Install Java'
-        if: ${{ github.base_ref != 'main' }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v4
         with:
-          java-version: 17
-      - name: 'Configure: Install GraalVM'
-        if: ${{ github.base_ref == 'main' }}
-        uses: graalvm/setup-graalvm@v1
-        with:
-          version: 'latest'
-          java-version: '17'
-          components: 'js'
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          java-version: '17.0.12'
```

Comment:
Any particular reason so specific w/ patch version?

Reply:
`setup-java@v4` requires complete version for some reason it doesn't select latest when you only specify `17`